deployment/helm/README.md

```diff
@@ -248,45 +248,55 @@ deployment:
     # - containerPort: 5555
     #   protocol: TCP
 
-    # Optional.  Rename the main root folder to something else.  For existing installs, this can be
-    # omitted.
-    # skahaTld: "/arc"
+    # Set the top-level-directory name that gets mounted at the root.
+    # skahaTld: "/cavern"
```

Comment:
I know it's not related to this PR, but I wonder if cavernRoot would have been a better name.
But also, what is this used for?  I know we're trying to decouple skaha and storage altogether, but I guess this field is used in the few references remaining?